### ISSUE TITLE
Glitches in IE11 after doing page refresh

### ISSUE THREAD

**Issue by jayudhandha** [quasarframework/quasar#2390]
### Software version

Quasar: v0.17.7
OS: Windows 7
Node: v9.4.0
NPM:5.6.0
Browsers: IE11

### What steps did you take, to get the error? Login into UI (IE11) and press F5 2-3 times.

I am attaching the actual UI and the glitches that i got after pressing F5.

**Original**:
![original](https://user-images.githubusercontent.com/5781316/43771222-1bee1cee-9a5c-11e8-8834-9ff30eda0431.PNG)

**Glitched**:
![error](https://user-images.githubusercontent.com/5781316/43771223-1c1a52dc-9a5c-11e8-96de-63f5ecacee91.PNG)

Any help?

**Comment by jayudhandha** [comment]
I don't know why its working fine with quasar v0.15.14 and IE11

**Comment by rstoenescu** [comment]
It has nothing to do with Quasar version...

**Comment by jayudhandha** [comment]
Yes, for older version its taking few seconds to load font files but at the end its getting loaded.
For newer version, if it's not getting loaded than it won't work till the end.

**Comment by rstoenescu** [comment]
I'm gonna repeat myself. But nothing changed in this regard... it's just font files that the browser itself handles.

**Comment by jayudhandha** [comment]
As my browser is not able to load this font files that i have used, 

How can i get these same icons that i have used without a material-icon library?

**Comment by jayudhandha** [comment]
I have changed the icon theme from material-icons to fontawesome.
Still a i am getting the same issue.
After doing page refresh few times, I am getting blank screen in place of fontawesome icons.

**Comment by jayudhandha** [comment]
@rstoenescu Finally i have found the solution.
I have load this material-icon from CDN in `index.template.html` and removed from `quasar.conf.js`

```
<!-- CDN example for Material Icons -->
  <link
    rel="stylesheet"
    href="https://fonts.googleapis.com/icon?family=Material+Icons"
  >
```

Thanks!
